google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/StorageSnippets.java

```diff
@@ -1157,6 +1152,21 @@ public void getBucketMetadata(String bucketName) throws StorageException {
     System.out.println("StorageClass: " + bucket.getStorageClass().name());
     System.out.println("TimeCreated: " + bucket.getCreateTime());
     System.out.println("VersioningEnabled: " + bucket.versioningEnabled());
+    String labelsString = null;
+    if(bucket.getLabels() != null) {
+      StringBuilder labels = new StringBuilder("{");
+      Iterator<Map.Entry<String, String>> iterator = bucket.getLabels().entrySet().iterator();
+      while (iterator.hasNext()) {
+        Map.Entry<String, String> next = iterator.next();
+        labels.append(next.getKey() + "=" + next.getValue());
+        if (iterator.hasNext()) {
+          labels.append(",");
+        }
+      }
+      labels.append("}");
+      labelsString = labels.toString();
+    }
+    System.out.println("Labels: " + labelsString);
```

Comment:
Could you follow this pattern instead? https://github.com/googleapis/google-cloud-java/blob/0731387f0edd9991dca3483aa2aa5a82ed96e0ac/google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/StorageSnippets.java#L1219-L1224

I misspoke, I should have pointed to this example first.